examples/http.c

```diff
@@ -26,17 +33,17 @@ static void response_cb(UtObject *object, UtObject *response) {
 
 int main(int argc, char **argv) {
   if (argc != 2) {
-    printf("Usage: http [uri]\n");
+    printf("Usage: http [url]\n");
```

Comment:
I think in this case it's a URL, not a URI (I'm always confused about the distinction between the two). In this case the param name is wrong in ut-http-client.c.

Reply:
Yeah it is a URI too. URL is a specific type of URI, i.e. it includes http scheme, host and port. URI is any kind of path resource, i.e. could refer to /some/file.bin, but yeah uri isn't incorrect, but URL is more specific for an http client :) I think I just typed it out of habbit.